packages/web-components/src/components/textarea/textarea.mdx

```diff
@@ -1,76 +1,52 @@
-import * as TextArea from './textarea.stories';
+import { ArgTypes, Canvas, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs } from '../../globals/internal/storybook-cdn';
+import * as TextAreaStories from './textarea.stories';
 
-<Meta of={TextArea} />
+<Meta of={TextAreaStories} />
 
-# Textarea
+# Text Area
 
-> 💡 Check our
-> [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/packages/web-components/examples/components/textarea)
-> example implementation.
+[Source code](https://github.com/carbon-design-system/carbon/tree/main/packages/web-components/src/components/text-area)
+&nbsp;|&nbsp;
+[Usage guidelines](https://www.carbondesignsystem.com/components/text-area/usage)
+&nbsp;|&nbsp;
+[Accessibility](https://www.carbondesignsystem.com/components/text-area/accessibility)
 
-[![Edit carbon-web-components](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/packages/web-components/examples/components/textarea)
+## Table of Contents
 
-Text inputs enable the user to interact with and input content and data. This
-component can be used for long and short form entries.
+- [Overview](#overview)
+- [Skeleton](#skeleton)
+- [AI Label](#ai-label)
+- [Component API](#component-api)
+- [CDN](#cdn)
+- [Feedback](#feedback)
 
-Textarea is a multi-line variant of text input.
+## Overview
 
-## Getting started
+<Canvas of={TextAreaStories.Default} />
 
-Here's a quick example to get you started.
-
-### JS (via import)
-
-```javascript
-import '@carbon/web-components/es/components/textarea/index.js';
-```
-
-<Markdown>{`${cdnJs({ components: ['textarea'] })}`}</Markdown>
-<Markdown>{`${cdnCss()}`}</Markdown>
-
-### HTML
+## Skeleton
 
-```html
-<cds-textarea placeholder="Optional placeholder text" rows="4" cols="50">
-  <span slot="label-text">Label text</span>
-  <span slot="helper-text">Optional helper text</span>
-  <span slot="validity-message">Something isn't right</span>
-</cds-textarea>
-```
+<Canvas of={TextAreaStories.skeleton} />
 
-## Form item
+## AI Label
 
-If the textarea is part of a form, make sure to import and wrap the textarea
-with `<cds-form-item>`
+<Canvas of={TextAreaStories.WithAILabel} />
 
-```javascript
-import '@carbon/web-components/es/components/form/index.js';
-```
+## Component API
 
-```html
-<cds-form-item>
-  <cds-textarea placeholder="Optional placeholder text" rows="4" cols="50">
-    <span slot="label-text">Label text</span>
-    <span slot="helper-text">Optional helper text</span>
-    <span slot="validity-message">Something isn't right</span>
-  </cds-textarea>
-</cds-form-item>
-```
+## `<cds-textarea>`
 
-## Skeleton
+<ArgTypes of="cds-textarea" />
 
-For the skeleton variation, utilize `<cds-textarea-skeleton>`.
+## `<cds-textarea-skeleton>`
 
-```html
-<cds-textarea-skeleton></cds-textarea-skeleton>
-```
+<ArgTypes of="cds-textarea-skeleton" />
```

Comment:
```suggestion
```

There's really no API for this, and the lone, no-description-having `styles` property is kind of confusing. Maybe we just ditch this one?

Reply:
deleted and removed others with the same only styles propery